.github/workflows/build-fdroid-apk.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         uses: gradle/actions/setup-gradle@v4
 
       - name: Assemble F-Droid release apk
+        env:
```

Comment:
Do we need to add it to f-droid apk build? (actually I'm not sure when we use this workflow 🤔)

Reply:
I'm not sure, because I don't think we use GHA for f-droid. Pinging @malmstein just in case

Reply:
We build the F-Droid APK for testing purposes only, and to validate that we don’t break anything on that flavour when merging things to develop. The actual F-Droid build happens in their Gitlab. 

For now we are still using Jenkins to submit changes, but it’s on the roadmap to also migrate it to GHA. It’s the last thing to move over.